structure/0006-governance.md

```diff
@@ -2,53 +2,152 @@
+
+## Processes
+
+To streamline working with teams, some standard processes need to be defined.
+The following processes apply to all teams unless that team's RFC specifies a different process.
+
+### Voting
+
+Unless specified otherwise, the following process is used for voting:
+
+A vote in a team may be called at any time by a team lead for that team or for one of its parent teams.
+A vote can also be initiated as required by another process, such as adding or removing a team member. 
+
+When a vote is called within a team, all members of that team and its parent teams are eligible to vote.
+All polls should include a positive and negative option, as well as the option to abstain.
+Voting members that abstain are excluded from percentages and tallies.
+Voting members not casting a vote within the allotted time frame are counted as abstained.
+
+After one week, the plurality of votes wins. The vote may also be called already if all eligible members 
+casted their vote.
+
+### Introduction of Staff
+
+Any community member is eligible to be added to a team by a majority vote of all eligible organization members.
+The vote is started on request from an eligible organization member. A community member wishing to join a team may
+kindly ask an eligible organization member to request a vote. A positive response from the organization member isn't
+mandatory.
+The following members are eligible:
+- All members of the relevant team
+- All members of all the sub teams of the relevant team
+- All members of all the parent teams of the relevant team
+
+During the vote, especially if the community member isn't already part of any team, the rest of the organization is
+consulted for opinions.
+
+### Removal of Staff
+
+Any team member or administrative board member may be removed by a majority vote of all eligible organization members.
+The following members are eligible:
+- All members of the relevant team
+- All members of all the sub teams of the relevant team
+- All members of all the parent teams of the relevant team
 
 # Drawbacks
-In the present day with most accounts and digital spaces being controlled primarily by one person, the idea of an "administrative board" has some downsides. A member of the board who is asked to step down may simply refuse to relinquish any resources they control. For this reason, personal character needs to be a high priority when electing members to the board so that such a situation can be avoided.
+
+In the present day with most accounts and digital spaces being controlled primarily by one person,
+the idea of an "administrative board" has some downsides.
+A member of the board who is asked to step down may simply refuse to relinquish any resources they control.
+For this reason, personal character needs to be a high priority when electing members to the board,
+so that such a situation can be avoided.
```

Comment:
There are solutions to this, and they should be discussed here, IMO - for example, the infra team controlling ClickUp as it currently does will mean that this can't happen there.

Admins aren't necessarily meant to be resource owners, they're a steering and oversight group - ownership of resources should go to the teams handling them, where that's actually a thing.

Reply:
With clickup you just move the issue to the infra team instead. That's not *really* a solution.

Having the admin team not being owners completely falls apart in many cases, but the discussion about Admin responsibilities is a separate one that probably doesn't go into this PR (althouh possibly into this RFC).